.github/workflows/ci.yml

```diff
@@ -67,7 +67,7 @@ jobs:
     strategy:
       matrix:
         xcode:
-          - /Applications/Xcode_13.3.1.app
+          - /Applications/Xcode_14.0.1.app
```

Comment:
We have to keep Xcode 13 until we drop support for it. 
The plugin code should be updated to compile on Xcode 13